setup.py

```diff
@@ -87,7 +87,8 @@ def get_requirements(requirements_file='requirements.txt'):
         'nose',
         'nose-cov',
         'webtest',
-        'mock'
+        'mock',
+        'diff-cover'
```

Comment:
In addition to this, let's also add ```python2-diff-cover``` [here](https://github.com/fedora-infra/bodhi/blob/2.6.2/devel/ansible/roles/dev/tasks/main.yml#L30) so that the RPM gets installed in the Vagrant box automatically.